Assignments/Program.cs

```diff
@@ -1,3 +1,65 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T17 - String Permutation
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Spark;
+public class T17 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "String Permutations Computer:-" + "\x1B[0m");
+      List<string> words = new () { "or", "not", "tat", "abcd", "abca" };
+      foreach (string word in words) PrintResult (word);
+      Console.WriteLine ();
+      Console.Write ("Now, let's try entering a word: ");
+      PrintResult (Console.ReadLine ());
+   }
+
+   /// <summary> Prints the permutated strings of given input word</summary>
+   public static void PrintResult (string input) {
+      if (input.Length > 7) Console.WriteLine ("Loading...");
+      Console.WriteLine ();
+      Console.WriteLine ($"Input word: {input}");
+      var answer = PermutationsOf (input);
+      int count = 0;
+      foreach (var item in answer) {
+         Console.Write ($"{++count,3}. ");
+         Console.ForegroundColor = ConsoleColor.Green;
+         Console.WriteLine (item);
+         Console.ResetColor ();
+      }
+   }
+
+   /// <summary>A recusive method which returns permutations of given input word</summary>

```

Comment:
```suggestion
   /// <summary>A recursive method which returns permutations of given input word</summary>

```
@vikrampg1710-tmm Check typos